samples/13.core-bot/src/main/java/com/microsoft/bot/sample/core/MainDialog.java

```diff
@@ -126,27 +140,34 @@ public MainDialog(FlightBookingRecognizer withLuisRecognizer, BookingDialog book
     }
 
     /**
-     * Shows a warning if the requested From or To cities are recognized as entities but they are not in the Airport entity list.
-     * In some cases LUIS will recognize the From and To composite entities as a valid cities but the From and To Airport values
+     * Shows a warning if the requested From or To cities are recognized as entities
+     *         but they are not in the Airport entity list.
+     * In some cases LUIS will recognize the From and To composite entities as a valid cities
+     *          but the From and To Airport values
      * will be empty if those entity values can't be mapped to a canonical item in the Airport.
      * @param turnContext A {@link WaterfallStepContext}
      * @param fromEntities An ObjectNode with the entities of From object
      * @param toEntities An ObjectNode with the entities of To object
      * @return A task
      */
-    private static CompletableFuture<Void> showWarningForUnsupportedCities(TurnContext turnContext, ObjectNode fromEntities, ObjectNode toEntities) {
+    private static CompletableFuture<Void> showWarningForUnsupportedCities(TurnContext turnContext,
+                                                                           ObjectNode fromEntities,
+                                                                           ObjectNode toEntities) {
         List<String> unsupportedCities = new ArrayList<String>();
 
-        if (StringUtils.isNotBlank(fromEntities.get("from").toString()) && StringUtils.isBlank(fromEntities.get("airport").toString())) {
+        if (StringUtils.isNotBlank(fromEntities.get("from").toString())
+            && StringUtils.isBlank(fromEntities.get("airport").toString())) {
             unsupportedCities.add(fromEntities.get("from").toString());
         }
 
-        if (StringUtils.isNotBlank(toEntities.get("to").toString()) && StringUtils.isBlank(toEntities.get("airport").toString())) {
+        if (StringUtils.isNotBlank(toEntities.get("to").toString())
+            && StringUtils.isBlank(toEntities.get("airport").toString())) {
```

Comment:
This should be `asText()` instead of `toString()`, right?

Reply:
I'll check it

Reply:
You're right. I'll change it